Gemfile.lock

```diff
@@ -482,10 +480,7 @@ DEPENDENCIES
   delayed_job
   delayed_job_active_record
   elasticsearch (~> 7.10.1)
```

Comment:
Do we have plans to remove this? I am guessing we can't upgrade our cluster as long as we have this in out gemfile.

Reply:
Yes, plan is to move to OpenSearch gems once this lands to production.

Reply:
Can you please elaborate on the order of changes?
- deploy this to prod
- replace elasticsearch in gemfile with opensearch (does opensearch gem work with ES cluster)
- upgrade ES cluster engine in staging to opensearch

Is this the plan?

Reply:
Once this will be merged, deployed to staging and considered safe to deploy to production, we can deploy to production.

After that my plan is to open other PR moving out of elasticsearch gem (migrate to opensearch). That can be merged and tested on staging as well. There is no need for upgrade of ES cluster.

And again after some time we can finally upgrade ES cluster engine. First I would like to try on staging to see if any downtime is needed and get some estimate on time needed for upgrade in general and repeat in production. If I understand it well, application works well without ES. The only problem could be unindexed changes. We can track gems recently changed (during maintenance window) and explicitly ask for reindex for those.

That's my initial plan.